src/lib/util/contenthandler.js

```diff
@@ -291,6 +343,28 @@ define([
 		}
 	}
 
+	/**
+	 * Check elements should be transformed.
+	 *
+	 * @returns true if elements should be transformed
+	 */
+	function isTransformFormattingsByMapping() {
+		if (
+			Aloha.settings.contentHandler &&
+			Aloha.settings.contentHandler.handler &&
+			Aloha.settings.contentHandler.handler.generic
+		) {
+			if (typeof Aloha.settings.contentHandler.handler.generic.transformFormattingsMapping === 'undefined' || 
```

Comment:
I would be a big fan of putting `Aloha.settings.contentHandler.handler.generic.transformFormattingsMapping` in a local variable, but admit that this ia personal preference :-).